cli/src/main/java/com/box/l10n/mojito/cli/command/checks/ContextAndCommentCliChecker.java

```diff
@@ -137,4 +148,8 @@ private boolean isPlural(AssetExtractorTextUnit assetExtractorTextUnit) {
   private boolean isBlank(String string) {
     return StringUtils.isBlank(string);
   }
+
+  public void setExcludedFilesPattern(Pattern excludedFilesPattern) {
+    this.excludedFilesPattern = excludedFilesPattern;
```

Comment:
I think we can drop this if we use the checker options object to pass the pattern. See https://github.com/pinterest/mojito/blob/master/cli/src/main/java/com/box/l10n/mojito/cli/command/checks/SpellCliChecker.java#L104 for an example of how it's used for the spell checker

Reply:
Done